configure.ac

```diff
@@ -117,6 +117,11 @@ AC_ARG_ENABLE(pixman, AS_HELP_STRING([--enable-pixman],
               [Use pixman library (default: no)]),
               [], [enable_pixman=no])
 AM_CONDITIONAL(XRDP_PIXMAN, [test x$enable_pixman = xyes])
+AM_CONDITIONAL(XRDP_PAINTER, [test x$enable_painter = xyes])
+AC_ARG_ENABLE(painter, AS_HELP_STRING([--enable-painter],
+              [Use painter library (default: no)]),
+              [], [enable_painter=no])
+AM_CONDITIONAL(XRDP_PAINTER, [test x$enable_painter = xyes])
```

Comment:
If there any need to support configurations both with and without libpainter?

Please add `AC_CONFIG_SUBDIRS([libpainter])` to make libpainter a subpackage. I tried that, it works much better than I expected.

Also, the top-level Makefile.am should have `libpainter` added to `SUBDIRS` above `xrdp` to compile `libpainter` before it's used.